cdc_rsync/cdc_rsync_client.cc

```diff
@@ -203,16 +213,32 @@ absl::Status CdcRsyncClient::StartServer(int port, const ServerArch& arch) {
     return MakeStatus(
         "Required instance component not found. Make sure the file "
         "%s resides in the same folder as %s.",
-        arch.CdcServerFilename(), kCdcRsyncFilename);
+        arch.CdcServerFilename(), ServerArch::CdcRsyncFilename());
   }
   std::string component_args = GameletComponent::ToCommandLineArgs(components);
-  std::string remote_command = arch.GetStartServerCommand(
-      kExitCodeNotFound, absl::StrFormat("%i %s", port, component_args));
-  ProcessStartInfo start_info =
-      remote_util_.BuildProcessStartInfoForSshPortForwardAndCommand(
-          port, port, /*reverse=*/false, remote_command);
+
+  ProcessStartInfo start_info;
   start_info.name = "cdc_rsync_server";
 
+  if (remote_util_) {
+    // Run cdc_rsync_server on the remote instance.
+    std::string remote_command = arch.GetStartServerCommand(
+        kExitCodeNotFound, absl::StrFormat("%i %s", port, component_args));
+    start_info = remote_util_->BuildProcessStartInfoForSshPortForwardAndCommand(
+        port, port, /*reverse=*/false, remote_command);
+  } else {
+    // Run cdc_rsync_server locally.
+    std::string exe_dir;
+    absl::Status status = path::GetExeDir(&exe_dir);
+    if (!status.ok()) {
```

Comment:
Why not just `RETURN_IF_ERROR(...) << "Failed to get exe directory"`?

Reply:
Done.